UI/Reports/balance_sheet.html

```diff
@@ -69,7 +69,7 @@
    END;
  ?>
 <tr <?lsmb class; IF report.rheads.ids.$row.props.account_number;
-                     'data-lsmb-account="';
+                     ' data-lsmb-account="';
```

Comment:
Why is there an extra space here?

Reply:
Because otherwise this tag get immediately concatenated to the previous and W3C complains about a missing separation between tags.

Reply:
Ah. I'm completely missing the "class" output in my review. Thanks for the explanation.